mm/kfence/report.c

```diff
@@ -114,10 +116,11 @@ static void dump_bytes_at(unsigned long addr)
 {
 	unsigned char *c = (unsigned char *)addr;
 	unsigned char *max_addr = (unsigned char *)min(ALIGN(addr, PAGE_SIZE), addr + 16);
-	pr_err("Bytes at %px:", (void *)addr);
+
+	pr_cont("[");
 	for (; c < max_addr; c++)
 		pr_cont(" %02X", *c);
-	pr_cont("\n");
+	pr_cont(" ]");
```

Comment:
Shall we bother printing the expected bytes as well?
I think people won't remember our pattern by heart (we may as well want to randomize it in the future).